docs/standard/architecture-containerized-liefecycle/docker-terminology.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Docker terminology | Microsoft Docs 
+description: Containerized Docker Application Lifecycle with Microsoft Platform and Tools
+keywords: Docker, Microservices, ASP.NET, Container
+author: CESARDELATORRE
+ms.author: wiwagn
+ms.date: 09/21/2017
+---
+# Docker terminology
+
+This section lists terms and definitions with which you should become familiar with before delving deeper into Docker (for further definitions, see the extensive [glossary](https://docs.docker.com/v1.11/engine/reference/glossary/) provided by Docker at <https://docs.docker.com/v1.11/engine/reference/glossary/>):
```

Comment:
This is an older url for the  glossary, the current one is here: https://docs.docker.com/glossary/, also it's listed twice with an extra parentheses at the end.